ReactiveObjCTests/RACMulticastConnectionSpec.m

```diff
@@ -45,15 +45,16 @@
 		expect(@(subscriptionCount)).to(equal(@1));
 	});
 
-	qck_it(@"shouldn't reconnect after disposal", ^{
```

Comment:
Yes it is intentional. But it shouldn't because it differs from the behavior of Rx (C#).